app/domain/logs.rb

```diff
@@ -360,6 +360,35 @@ module AuthnJwt
         code: "CONJ00070D"
       )
 
+      FetchingJwtConfigurationValue = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching jwt signing key uri from '{0-resource-id}'",
```

Comment:
Fetching JWT signing key uri from '{0-resource-id}'"

(not sure what this means, might change when I know better.)

Reply:
It fetches it from authenticator configuration.
Changed to 
`"Fetching JWT signing key URI from authenticator configuration'{0-resource-id}'..."`